sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -366,14 +375,138 @@ CosmosAsyncContainer setId(String id) {
     }
 
     /**
-     * Gets a CosmosAsyncItem object without making a service call
+     * Reads an item.
+     * <p>
+     * After subscription the operation will be performed.
+     * The {@link Mono} upon successful completion will contain a cosmos item response with the read item
+     * In case of failure the {@link Mono} will error.
+     *
+     * @param itemId the item id
+     * @param partitionKey the partition key
+     * @return an {@link Mono} containing the cosmos item response with the read item or an error
```

Comment:
3rd `@param` is missing. 
Please replace `an` with `a` in return statement before `Mono`. 

Please do this change here and everywhere else.

Reply:
Done